packages/manager/apps/web/client/app/app.js

```diff
@@ -175,6 +173,11 @@ angular
       $qProvider.errorOnUnhandledRejections(false);
     },
   ])
+  .config(/* @ngInject */ (ovhPaymentMethodProvider) => {
+    ovhPaymentMethodProvider.setPaymentMethodPageUrl(
```

Comment:
I wonder if we should handle this in `@ovh-ux/manager-core` instead

Reply:
When I suggested we initialize our dependencies in `@ovh-ux/manager-core` I was told it is a bad practice so I guess we shouldn't do that

Reply:
@FredericEspiau how come ? We use `manager-core` to initialize translations settings because there are common. I found this practice better to duplication in that case. 
I suppose it would depend on the context and the use and there is no strict rule on whether it should or should not be in a common module, but that's my opinion.

Reply:
@marie-j I agree, we could transfer this in `@ovh-ux/manager-core` as all the manager apps will have the same url for this payment method page.